customize.dist/src/less2/include/dropdown.less

```diff
@@ -160,12 +160,24 @@
             li[role="menuitem"] {
                 border-radius: @variables_radius;
                 white-space: nowrap;
-                &:hover {
-                    background-color: @cp_dropdown-bg-hover !important;
-                }
                 &:focus-visible {
                     outline-color: @cp_dropdown-fg;
                 }
+                &:hover {
+                    background-color: @cp_dropdown-bg-hover;
+                }
+            }
+            li[role="menuitem"].cp-notification-avatar {
```

Comment:
`dropdown.less` is a generic file containing style for all dropdown in CryptPad. You shouldn't put style specific to the notifications here. `notifications.less` is more appropriate for these changes.